gtsfm/data_association/point3d_initializer.py

```diff
@@ -70,9 +118,7 @@ class Point3dInitializer(NamedTuple):
     """
 
     track_camera_dict: Dict[int, PinholeCameraCal3Bundler]
-    mode: TriangulationParam
-    reproj_error_thresh: float
-    num_ransac_hypotheses: Optional[int] = None
+    options: TriangulationOptions
```

Comment:
can we rename to triangulation_options to prevent ambiguity?

Reply:
I feel like maybe that would be too wordy/long. Just my two cents.

Reply:
I think just `options` is sufficient since there aren't any similar attributes.